pkg/southbound/synchronizer/factory_test.go

```diff
@@ -16,9 +16,9 @@ package synchronizer
 
 import (
 	devicechange "github.com/onosproject/onos-config/api/types/change/device"
-	"github.com/onosproject/onos-config/pkg/dispatcher"
+	dispatcherpkg "github.com/onosproject/onos-config/pkg/dispatcher"
```

Comment:
why did you go with the pkg ?

Reply:
I suspect this is to disambiguate the package 'dispatcher' from the object reference 'dispatcher'.

Reply:
@SeanCondon is correct. There were clashes between variable names and the import name.